hack/release.sh

```diff
@@ -344,18 +348,23 @@ if [ "${PUSH}" ]; then
   if [ "${REGISTRY}" ]; then
     CSI_IMAGE_NAME="${REGISTRY}driver"
   fi
-  # build windows images and push them to registry as currently windows images are build only when push is enabled
-  WINDOWS_IMAGE_OUTPUT="type=registry"
-  for osversion in "${OSVERSION_WIN[@]}"
-  do 
-    OSVERSION="$osversion"
-    build_driver_images_windows 
-  done
+
+  if [ "$DO_WINDOWS_BUILD" = true ]; then
+    # build windows images and push them to registry as currently windows images are build only when push is enabled
+    WINDOWS_IMAGE_OUTPUT="type=registry"
+    for osversion in "${OSVERSION_WIN[@]}"
+    do
+      OSVERSION="$osversion"
+      build_driver_images_windows
+    done
+  fi
   # tag linux images with linux and push them to registry
   LINUX_IMAGE_OUTPUT="type=registry"
   build_driver_images_linux
-  #create and push manifest for driver
-  push_manifest_driver
+  if [ "$DO_WINDOWS_BUILD" = true ]; then
+    #create and push manifest for driver
+    push_manifest_driver
```

Comment:
Is this a windows node specific function?

Reply:
we added this manifest file to publish information about different platforms. Now that we are skipping the windows platform, I thought it should be fine to skip creating the docker image manifest file.